letsencrypt/translations/en.yaml

```
@@ -37,5 +32,14 @@ configuration:
   dns:
     name: DNS
     description: DNS Provider configuration
+  keytype:
+    name: Key Type
+    description: >-
+      Select the certificate key type. Leave this unset for auto-detect or
+      choose a key type to override the default behavior.
```

It is kinda obvious that setting it will force it, but the default might be worth noting here:

```suggestion
      Select the certificate key type. If unset, will auto-detect based on
      the key type of the existing certificate or default to ecdsa.
```